expan/core/experiment.py

```diff
@@ -190,18 +190,16 @@ def analyze_statistical_test_suite(self, test_suite, test_method='fixed_horizon'
         return test_suite_result
 
 
-    def outlier_filter(self, data, kpis, percentile=99.0, threshold_type='upper'):
+    def outlier_filter(self, data, kpis, thresholds=None):
         """ Method that filters out entities whose KPIs exceed the value at a given percentile.
         If any of the KPIs exceeds its threshold the entity is filtered out. 
```

Comment:
This changes the interface and therefore will break existing code for some users (i.e. us!).

Do we care about this? @shansfolder @daryadedik 

We could make changes to ensure compatibility

Reply:
The other parts of the highly classified Z. code should be changed accordingly, but fortunately, it's just one call.

Reply:
I'll 'resolve' this. It's true that we don't need to ensure compatibility here